test/ILLink.Tasks.IntegrationTests/TestContext.cs

```diff
@@ -74,9 +74,15 @@ public static void SetupDefaultContext()
 			Configuration = "Release";
```

Comment:
This can be removed

Reply:
I'm trying to separate the illink configuration from the test project config. My intention here is to publish+link test projects in Release, using either Debug or Release illink.

Reply:
Got it - thanks.